cranelift/codegen/src/isa/s390x/abi.rs

```diff
@@ -509,28 +509,26 @@ impl ABIMachineSpec for S390xMachineDeps {
         outgoing_args_size: u32,
     ) -> (u64, SmallVec<[Inst; 16]>) {
         let mut insts = SmallVec::new();
-        let mut clobbered_fpr = vec![];
-        let mut clobbered_gpr = vec![];
 
-        for &reg in clobbered_callee_saves.iter() {
-            match reg.to_reg().class() {
-                RegClass::Int => clobbered_gpr.push(reg),
-                RegClass::Float => clobbered_fpr.push(reg),
-            }
-        }
+        // Collect clobbered registers.
+        let mut clobbered_fpr = vec![];
+        let mut first_clobbered_gpr = 16;
```

Comment:
Could we add a comment here describing the invariant? I.e. that registers `first_clobbered_gpr..16` are clobbered (a contiguous range)?

(I realize the variable and logic are pre-existing but it's good to clarify while we're passing through :-) )